doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1429,7 +1491,8 @@ An `http` object contains the following subfields:
   compliant server must support `http` URL schema, other schemas could also be supported.
 + `headers` (optional): a key-map of HTTP headers that are included in the notification messages.
 + `qs` (optional): a key-map of URL query parameters that are included in the notification message.
-+ `method` (optional): the method to use for sending the notification (default is POST)
++ `method` (optional): the method to use for sending the notification (default is POST). Only valid
+  HTTP methods are allowed.
```

Comment:
The subscription-creation would fail.
Was this a question or do you mean that this information should be a part of the spec?